packages/scene-composer/src/components/panels/scene-components/data-overlay/__snapshots__/DataBindingMapEditorSnap.spec.tsx.snap

```diff
@@ -0,0 +1,379 @@
+        label="Binding Name"
+      >
+        <div
+          data-mocked="Input"
+          data-test-id="binding-name-input"
+          value="binding-1"
+        />
+      </div>
+      <div
+        data-mocked="Box"
+        margin="{\\"vertical\\":\\"s\\"}"
+      >
+        <div
+          data-mocked="SpaceBetween"
+        >
+          <div
+            data-mocked="FormField"
+            label="Entity Id"
+          >
+            <div
+              data-mocked="Autosuggest"
+              options="[{\\"label\\":\\"label1\\",\\"value\\":\\"value1\\"},{\\"label\\":\\"label2\\",\\"value\\":\\"value2\\"}]"
+              value="value1"
+            />
+          </div>
+          <div
+            data-mocked="FormField"
+            label="Component Type Id"
+          >
+            <div
+              data-mocked="Select"
+              data-testid="value-data-binding-builder-select"
+              options="[{\\"label\\":\\"label1\\",\\"value\\":\\"value1\\"},{\\"label\\":\\"label2\\",\\"value\\":\\"value2\\"}]"
+              placeholder="Select an option"
+              selectedoption="null"
+            />
+          </div>
+        </div>
+      </div>
+      <div
+        data-mocked="Button"
+        data-test-id="remove-binding-button"
+      >
+        Remove data binding
+      </div>
+    </div>
+    <div
+      data-mocked="ExpandableSection"
+      header="binding-2"
+    >
+      <div
+        data-mocked="FormField"
+        errortext="null"
+        label="Binding Name"
+      >
+        <div
+          data-mocked="Input"
+          data-test-id="binding-name-input"
+          value="binding-2"
+        />
+      </div>
+      <div
+        data-mocked="Box"
+        margin="{\\"vertical\\":\\"s\\"}"
+      >
+        <div
+          data-mocked="SpaceBetween"
+        >
+          <div
+            data-mocked="FormField"
+            label="Entity Id"
+          >
+            <div
+              data-mocked="Autosuggest"
+              options="[{\\"label\\":\\"label1\\",\\"value\\":\\"value1\\"},{\\"label\\":\\"label2\\",\\"value\\":\\"value2\\"}]"
+              value="value1"
+            />
+          </div>
+          <div
+            data-mocked="FormField"
+            label="Component Type Id"
+          >
+            <div
+              data-mocked="Select"
+              data-testid="value-data-binding-builder-select"
+              options="[{\\"label\\":\\"label1\\",\\"value\\":\\"value1\\"},{\\"label\\":\\"label2\\",\\"value\\":\\"value2\\"}]"
+              placeholder="Select an option"
+              selectedoption="null"
+            />
+          </div>
+        </div>
+      </div>
+      <div
+        data-mocked="Button"
+        data-test-id="remove-binding-button"
+      >
+        Remove data binding
+      </div>
+    </div>
+    <div
+      data-mocked="Button"
+      data-test-id="add-binding-button"
+    >
+      Add data binding
+    </div>
+  </div>
+</div>
+`;
```

Comment:
This is exactly my point from above, even with mocks, there are multiple fields with various data embedded, and 120 lines of DOM in the snapshot. This means this test can break for a lot of reasons, adding a new field, changing layout, etc. And not just one test (the snapshot), but all the tests will fail for these reasons.